script.js

```diff
@@ -1 +1,25 @@
-console.log('this works');
\ No newline at end of file
+// Set 1
+
+var div;
+
+// create a buildBox function
+function buildBox() {
+	// create and insert an empty div on the page with: positioning, top and left coordinates, background color expressed as rgba 
+	div = document.createElement('div');
+	div.style.cssText = 'width: 300px; height: 300px; position: absolute; top: 0; left: 0; background-color: rgba(0,0,0,1);';
+	document.body.appendChild(div);
+}
+
+// write a function "randInRange" that generates a random integer, given a range of possible values (min & max)
+function randInRange(min, max) {
+	return Math.floor(Math.random()*(max-min+1))+min;
```

Comment:
This is not very readable, and I think the math won't quite come out right for stuff near 0. Try to refactor this, perhaps by calculating the `range` before diving into `Math.random()`. Consider also inputs less than zero.
As a bonus, check that min and max were supplied in the right order. :)